content/posts/rsoc-2021-announcement/index.md

```diff
@@ -0,0 +1,47 @@
+---
+author: "xvilka"
+title: "Rizin Summer of Code 2021 Announcement"
+date: "2021-05-13"
+summary: "An announcement of the Rizin Summer of Code 2021. Two accepted candidates."
+tags: ["rizin", "rsoc"]
+ShowToc: true
+TocOpen: false
+weight: 2
+---
+
+For many years during our times at radare2 we participated in Google Summer of Code as an organization.
+Also, in addition to hosting those we often did our own Summer of Code. To continue the tradition, albeit under a
+completely new name and a few differences we introduce the RSoC 2021.
+[Together with the KeenLab of Tencent](https://keenlab.tencent.com/zh/2021/03/04/RSoC-keenlab-2021/) we provide an opportunity for students
+to apply for the internship and to work on improving Rizin. The application period
+continued through all of April with fierce competition between candidates. Finally, we chose
+two. Let them introduce themselves, and we wish them luck!
```

Comment:
I would go for another structure -
Starting with "We are excited to announce RSoC 2021. Rizin summer of code is... Together with the KeenLab of Tencent.... we provide the opportunity...."
Then "We use the experience we gathered by participating in Google Summer of Code as an organization and use it to...."
And then the closure sentences about the process and selected students

Reply:
Good suggestion! Implemented it.